src/ralph/deployment/tests/functional/tests_api.py

```diff
@@ -68,7 +76,7 @@ class TestVMCreation(UserTestCase):
 
     fixtures=['vm_creation_setup']
 
-    def test_correct(self): 
+    def test_correct(self):
```

Comment:
All tests: 

Comply with test naming : test_[object]_should_[action]_when[something]

put it in the name instead of docstrings